Solutions/ImpervaCloudWAF/Data Connectors/ImpervaWAFCloud_FunctionApp.json

```diff
@@ -0,0 +1,121 @@
+{
+	"id": "ImpervaWAFCloudAPI",
+	"title": "Imperva Cloud WAF",
+	"publisher": "Atlassian",
+	"descriptionMarkdown": "The [Imperva Cloud WAF](https://www.imperva.com/resources/resource-library/datasheets/imperva-cloud-waf/) data connector provides the capability integrate and to ingest Web Application Firewall events into Azure Sentinel through the REST API. Refer to Log integration [documentation](https://docs.imperva.com/bundle/cloud-application-security/page/settings/log-integration.htm#Download) for more information. The connector provides ability to get events which helps to examine potential security risks, analyze your team's use of collaboration, diagnose configuration problems and more.",
```

Comment:
Shouldn't the description text start be like ,

The Imperva Cloud WAF data connector provides the capability **to** integrate and ~~to~~ ingest Web Application Firewall events into Azure Sentinel through the REST API.

Reply:
done